docs/api.asciidoc

```diff
@@ -281,6 +281,36 @@ Returns the set `value`.
 
 WARNING: Be aware that tags are indexed in Elasticsearch. Using too many unique keys will result in *https://www.elastic.co/blog/found-crash-elasticsearch#mapping-explosion[Mapping explosion]*.
 
+NOTE: This method has been deprecated in favor of `set_label`, which does not convert values to Strings.
+
+[float]
+[[api-agent-set-label]]
+==== `ElasticAPM.set_label`
+
+Add a label to the current transaction.
+Labels are equivalent to what were previously called `tags`. They are basic key-value pairs that
```

Comment:
I'm not sure we need the historical reference here. Users just want to know what they can do now and probably don't care much about how things used to be? (I could be wrong?)

Reply:
Sure, I'll take out the overexplanation = )